src/Microsoft.DotNet.Tools.Compiler/Program.cs

```diff
@@ -289,7 +274,7 @@ private static bool CompileProject(ProjectContext context, CompilerCommandApp ar
             var compilerName = CompilerUtil.ResolveCompilerName(context);
 
             // Write RSP file
-            var rsp = Path.Combine(intermediateOutputPath, $"dotnet-compile.{context.ProjectFile.Name}.rsp");
+            var rsp = Path.Combine(intermediateOutputPath, $"dotnet-compile.rsp");
```

Comment:
I had projects stomping on the common csc response file when tests were executing in parallel. Could be gone now though. I had fixed it by manually disambiguating the interm path in the test code.